allennlp/data/tokenizers/pretrained_transformer_tokenizer.py

```diff
@@ -182,17 +183,43 @@ def tokenize(self, text: str) -> List[Token]:
         """
         return self._tokenize(text)
 
-    def intra_word_tokenize(self, tokens: List[str]) -> Tuple[List[Token], List[Tuple[int, int]]]:
+    def intra_word_tokenize(self, tokens_a: List[str], tokens_b: Optional[List[str]] = None) -> Any:
```

Comment:
Honestly, I'd suggest just making two separate functions `intra_word_tokenize` and `intra_word_tokenize_pair`. The calling code has to be different anyway to account for the different return type, so we might as well have two functions with much simpler type signatures.